.github/workflows/snapshot-release.yml

```diff
@@ -1,5 +1,6 @@
 name: Snapshot Release
 on:
+  workflow_dispatch:
```

Comment:
This workflow can now also be triggered from the GitHub UI, which has the benefit that it will actually run the workflow version of that branch and not the workflow from `main` - meaning, we can not test it before merging into `main`.